SmartDeviceLink/SDLTouchManager.m

```diff
@@ -141,39 +153,42 @@ - (void)sdl_handleTouchBegan:(SDLTouch *)touch {
     _performingTouchType = SDLPerformingTouchTypeSingleTouch;
 
     switch (touch.identifier) {
-        case SDLTouchIdentifierFirstFinger:
+            case SDLTouchIdentifierFirstFinger:
```

Comment:
This indentation for the whole switch is incorrect, should look like:

```objc
switch (touch.identifier) {
    case SDLTouchIdentifierFirstFinger: {
        self.previousTouch = touch;
    } break;
    case SDLTouchIdentifierSecondFinger: {
        _performingTouchType = SDLPerformingTouchTypeMultiTouch;
        self.currentPinchGesture = [[SDLPinchGesture alloc] initWithFirstTouch:self.previousTouch secondTouch:touch];
        self.previousPinchDistance = self.currentPinchGesture.distance;
        if ([self.touchEventDelegate respondsToSelector:@selector(touchManager:pinchDidStartAtCenterPoint:)]) {
            [self.touchEventDelegate touchManager:self pinchDidStartAtCenterPoint:self.currentPinchGesture.center];
        }
    } break;
}
```

Reply:
Fixed